app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
do you need to have all of these devise views in the application?
You only need them if you are going to modify them

Reply:
removed